### DISCUSSION TITLE
Using 24h time instead of am/pm

### DISCUSSION THREAD

**Discussion by Esther-00** [cortexrd/Knack-Toolkit-Library#347] [Q&A] [answered]
I'm new to using the KTL and want to implement it in my app. 
However the default settings change the time input to am/pm values, but I want to keep using 24h time. 
My app currently uses military time (24h) on all time fields, but the KTL overrides this setting. 

I don't see an option to change this in the KTL configurator.
If anyone knows a how to do this has a script, it would greatly appreciated if you go share it with me.

Any and all help on this would be appreciated… 

Esther

**Comment by CSWinnall** [comment]
Hi Esther

Welcome to the forum.  I also noticed this a couple of weeks ago but didn't think that it was the KTL.  After checking you are right the KTL does appear to be editing the format of the time to am/pm.  I believe Norm is away at the moment but he should be able to fix this when he gets back.  In the meantime I will try and find the code that is doing this so he'll be able to fix it quickly.
Thank you for letting us know about this.

Craig

**Reply by Esther-00** [reply]
Hi Craig, 

Thank you for looking into this and forwarding it to Norm. 
Thanks as well for the video tutorials you both made on the KLT, they have been very helpful!

**Reply by cortexrd** [reply]
Hi guys,

I'm back!

@Esther-00 welcome to the KTL and the forum.

I will investigate this in issue #349.  See you there.
Norm

**Comment by Deanobeano2** [comment]
Hi all

I just noticed random behaviour with this too. I entered a transaction and it was 24h time as it should have been and then the next transaction after that had pm for the time.

Dean

**Comment by cortexrd** [comment, marked as answer]
Hi to all,

This is the same as issue #349 and has been fixed in v0.28.6.
If all ok, someone mark this as resolved (Answered).

Norm
